messaging/src/main/java/org/axonframework/deadline/dbscheduler/DbSchedulerDeadlineManager.java

```diff
@@ -561,6 +567,19 @@ public Builder stopScheduler(boolean stopScheduler) {
             return this;
         }
 
+        /**
+         * Sets the {@link MessageNameResolver} used to resolve the {@link QualifiedName} when publishing {@link EventMessage EventMessages}.
```

Comment:
Darn, not sure if I suggested this wrongly now, but regardless, the JavaDoc should be about `DeadlinMessages`, not `EventMessages` :-[